index.bs

```diff
@@ -64,13 +64,6 @@ spec: TokenBinding; urlPrefix: https://tools.ietf.org/html/draft-ietf-tokbind-pr
         text: Token Binding
         text: Token Binding ID; url: section-3.2
 
-spec: WebCryptoAPI; urlPrefix: https://www.w3.org/TR/WebCryptoAPI/
-    type: dfn
-        text: normalizing an algorithm; url: dfn-normalize-an-algorithm
-        text: recognized algorithm name
-    type: dictionary
-        text: AlgorithmIdentifier; url: dfn-AlgorithmIdentifier
-
```

Comment:
when one deletes stuff like this, one needs to run BikeShed and check for errors  -- in this case, we still use the "recognized algorithm name" from WebCrypto, and that is causing linking errors without at least that portion the above in the spec.